packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -52,14 +75,18 @@ export const computeRoundTiming = (params, baseTime) => {
   const steps = TimeMath.divideRelRel(targetDuration, clockStep);
```

Comment:
Consider a refactoring of the intermediate computations as suggested in https://github.com/Agoric/agoric-private/issues/50#issuecomment-1547020495